core/server/data/schema/schema.js

```diff
@@ -172,6 +172,7 @@ module.exports = {
         slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
         secret: {type: 'string', maxlength: 150, nullable: false},
         redirection_uri: {type: 'string', maxlength: 2000, nullable: true},
+        blog_uri: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
Ok, I see where you are coming from. However, this field still doesn't feel right to me.

All of the other clients e.g. ghost-frontend are things that allow 3rd parties to talk to Ghost. 

This is the first client we have introduced that lets Ghost talk to a 3rd party. 

We will have more things like this, but none of them will have a "blog_uri", they may have another field.

If you look at reference material for similar systems: e.g. twitter's create an application, or GitHub, or Google API console, can you perhaps find a way to make this field work for more general cases than just this one?